pyEpiabm/pyEpiabm/core/household.py

```diff
@@ -65,3 +68,29 @@ def add_person(self, person):
         """
         self.persons.append(person)
         person.household = self
+        if person.infection_status == InfectionStatus.Susceptible:
+            self.add_susceptible_person(person)
+
+    def add_susceptible_person(self, susceptible_person):
+        """Adds a person to the list of susceptible people in the household.
```

Comment:
```suggestion
        """Adds a person to the set of susceptible people in the household.
```

Reply:
Changed back so the change is no longer needed